### ISSUE TITLE
Utiliser le même gris pour les croix de fermeture et flèches déroulantes

### ISSUE THREAD

**Issue by AlexSImon44** [irplab/irpia#107]
Utiliser le même gris pour les croix de fermeture et flèches déroulantes partout (elles sont actuellement de 2 gris différents).

**Comment by jdpro** [comment]
Je ne vois pas de croix de fermeture. Les flèches des menus déroulants (type de contribution, SIREN/ISNI, type documentaire, type pédagogique, domaine, niveau) me semblent noires.

**Comment by AlexSImon44** [comment]
Les flèches de la page contribution sont dans un gris (foncé).
<img width="863" alt="Capture d’écran 2022-09-26 à 17 09 39" src="https://user-images.githubusercontent.com/112937067/192313453-09d8c9ea-7243-4e2e-a074-7f60fa0df42c.png">

**Comment by jdpro** [comment]
Quand tu parles des croix de fermeture ce sont les boutons "X" pour viderle contenu d'un champ texte qui sont à l'étape suivante (description) ?

**Comment by AlexSImon44** [comment]
Non, je parle des flèches de fermeture des menus déroulants, elles sont grises foncé ! :)

**Comment by jdpro** [comment]
Je suis désolé je ne vois pas de flèches de fermetures de menus déroulants. Peut-être que c'est un contrôle natif qui n'apparaît que sur Mac ?

**Comment by AlexSImon44** [comment]
Je parle de la petite flèche noire au bout à droite du champs texte, comme ci-dessous :
<img width="824" alt="Capture d’écran 2022-10-04 à 10 16 21" src="https://user-images.githubusercontent.com/112937067/193769486-bac138eb-50e3-4fd2-b0f2-f45b6a2a45ac.png">
